### PR TITLE
Add discard draft and update draft

### PR DESCRIPTION
**Pull request by rilut**
- Append ":draft" to botId if tag=="draft" in node console
- Compile .ts into .js

### CODE REVIEW

**Comment by rilut** [comment]
@aliarham11 I request your review, please 🙏

**Comment by aliarham11** [comment]
still conflict @rilut

**Comment by rilut** [comment]
hold my beer

File: components/bots/bot.ts [hunk 1]
```diff
@@ -230,6 +235,7 @@ export default class Bot extends Component {
 
             try {
                 const result = await this.helper.toPromise(this.api.botApi, this.api.botApi.botsPost, botDesc);
+                desc.tag = "latest";
```
Comment (aliarham11) [inline components/bots/bot.ts]: Because user can have custom tag, please refer to line  223

File: components/bots/bot.ts [hunk 2]
```diff
@@ -245,7 +251,7 @@ export default class Bot extends Component {
         } else {
             try {
                 const result = await this.helper.toPromise(this.api.botApi, this.api.botApi.botsBotIdPut, botDesc.id, botDesc, {});
-
+                desc.tag = "latest";
```
Comment (aliarham11) [inline components/bots/bot.ts]: This too

**Pull request merged**